Common subdirectories: ../misc/model_generation/raw_generated/__pycache__ and ../grafana_dashboard/model/__pycache__
diff -Nau ../misc/model_generation/raw_generated/dashboard_types_gen.py ../grafana_dashboard/model/dashboard_types_gen.py
--- ../misc/model_generation/raw_generated/dashboard_types_gen.py	2023-07-11 13:22:31.000000000 +0800
+++ ../grafana_dashboard/model/dashboard_types_gen.py	2023-07-11 13:46:10.000000000 +0800
@@ -8,6 +8,9 @@
 
 from pydantic import BaseModel, Field, conint, constr
 
+from grafana_dashboard.model.lokidataquery_types_gen import LokiDataQuery
+from grafana_dashboard.model.prometheusdataquery_types_gen import PrometheusDataQuery
+
 
 class AnnotationPanelFilter(BaseModel):
     exclude: Optional[bool] = Field(
@@ -180,8 +183,9 @@
     empty = 'empty'
 
 
-class Target(BaseModel):
-    pass
+# NOTE REMOVED
+# class Target(BaseModel):
+#     pass
 
 
 class Threshold(BaseModel):
@@ -534,7 +538,7 @@
         description='FIXME this almost certainly has to be changed in favor of scuemata versions',
     )
     tags: Optional[List[str]] = Field(None, description='TODO docs')
-    targets: Optional[List[Target]] = Field(None, description='TODO docs')
+    targets: Optional[List[Union[PrometheusDataQuery, LokiDataQuery, Dict]]] = Field(None)  # NOTE MODIFIED
     title: Optional[str] = Field(None, description='Panel title.')
     description: Optional[str] = Field(None, description='Description.')
     transparent: Optional[bool] = Field(
